editor/src/components/canvas/commands/convert-inline-block-command.ts

```diff
@@ -0,0 +1,49 @@
+import { emptyComments, jsxAttributeValue } from '../../../core/shared/element-template'
+import { ValueAtPath } from '../../../core/shared/jsx-attributes'
+import { ElementPath } from '../../../core/shared/project-file-types'
+import { EditorState } from '../../editor/store/editor-state'
+import { stylePropPathMappingFn } from '../../inspector/common/property-path-hooks'
+import { applyValuesAtPath } from './adjust-number-command'
+import { BaseCommand, CommandFunction, WhenToRun } from './commands'
+
+export interface ConvertInlineBlock extends BaseCommand {
```

Comment:
my PR introduces a generic set property command – is there a merit to having this display prop specific one?